jobs/config.json

```diff
@@ -1970,7 +2165,8 @@
       "--timeout=180m",
       "--extract=ci/latest-1.7",
       "--check-leaked-resources",
-      "--test_args=--ginkgo.focus=\\[Feature:GPU\\]"
+      "--test_args=--ginkgo.focus=\\[Feature:GPU\\]",
+      "--gcp-project=k8s-jkns-e2e-gce-gpus-1-7"
```

Comment:
Actually there's a test which check for the existence of this flag, so don't remove it.